src/k8s/pkg/k8sd/features/contour/ingress.go

```diff
@@ -40,13 +40,25 @@ func ApplyIngress(ctx context.Context, snap snap.Snap, ingress types.Ingress, _
 	values = map[string]any{
 		"envoy-service-namespace": "projectcontour",
 		"envoy-service-name":      "envoy",
+		"envoy": map[string]any{
+			"image": map[string]any{
+				"registry":   registry,
```

Comment:
can you check if we can avoid setting `registry`, then we can just have `envoyImageRepo` be the full image ?

Reply:
Hm... Not sure if there is a nice way here.
Registry has a default value. We could try pass "" to registry and pack it in the repository but that feels a little wrong?
```
# values.yaml
  image:
    registry: docker.io
    repository: bitnami/contour
    tag: 1.28.2-debian-12-r4
```
Alternatively, there is a global image registry we could set:
```
global:
  imageRegistry: ""
```

Reply:
My suggestion is to try to see if it works. If it does not, then take the registry out of all images, and adjust the code accordingly

Reply:
Alright, setting this works:
```
  image:
    registry: ""
    repository: ghcr.io/canonical/k8s-snap/bitnami/contour
    tag: 1.28.2-debian-12-r4
    ```